tools/swift-inspect/Package.swift

```diff
@@ -9,7 +9,7 @@ let package = Package(
         .library(name: "SwiftInspectClient", type: .dynamic, targets: ["SwiftInspectClient"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-argument-parser", from: "0.0.1"),
+        .package(url: "https://github.com/apple/swift-argument-parser", from: "1.2.2"),
```

Comment:
This matches https://github.com/apple/swift/blob/main/utils/update_checkout/update-checkout-config.json#L103 which seems like a good idea.